cmd/torpedo/torpedo.go

```diff
@@ -325,6 +325,39 @@ func (t *torpedo) testNodeReboot(allNodes bool) error {
 	return err
 }
 
+// testResourceCleanupOnTearDown checks if all the resources used by the application
+// have been cleaned up
+func (t *torpedo) testResourceCleanupOnTearDown() error {
```

Comment:
Let's not add a new test. Resource cleanup would normally be default check for all tests and we might eventually do that if we have resources. So having a dedicated test for checking resource cleanup wouldn't be right. Rather, have one or more of the existing tests use the new functionality by invoking destroy with the new options.